_projects/2024/100540803/100540803.Rmd

```diff
@@ -0,0 +1,601 @@
+    axis.text = element_text(size = 12),
+    legend.position = "right",
+    legend.title = element_text(size = 10, face = "bold"),
+    legend.text = element_text(size = 9)
+  )
+plotA
+```
+
+The PlotB code uses `ggplot2` to create a faceted stacked bar chart that illustrates the changes in fish stock statuses across different ocean regions and time periods. The data is first classified into four statuses (e.g., "Healthy") based on `B_Bmsy` and `F_Fmsy`, then plotted as bars grouped by time periods and faceted by ocean regions.
+
+The chart uses a color palette to distinguish statuses, adds a dashed line for emphasis, and applies styling for clarity, effectively visualizing the trends in fish stock statuses over time and across regions.
+
+```{r}
+grouped_data <- grouped_data %>%
+  mutate(
+    status = case_when(
+      B_Bmsy < 1 & F_Fmsy > 1 ~ "Overfished & Overfishing",
+      B_Bmsy >= 1 & F_Fmsy > 1 ~ "Not Overfished & Overfishing",
+      B_Bmsy < 1 & F_Fmsy <= 1 ~ "Overfished but Not Overfishing",
+      B_Bmsy >= 1 & F_Fmsy <= 1 ~ "Healthy"
+    )
+  )
+plotB <- ggplot(grouped_data, aes(x = year_range, fill = status)) +
+  geom_bar(position = "stack", color = "white", width = 0.7) +
+  facet_wrap(~Ocean, scales = "free_x") +
+  scale_fill_brewer(palette = "Set2") +
+  annotate("segment", x = 5, xend = 5, y = 0, yend = Inf,
+           linetype = "dashed", color = "black", size = 1) +
+  labs(
+    title = " ",
+    subtitle = "Comparing Fish Stocks Status Across Oceans Over Time",
+    x = "Year Range",
+    y = "Count of Fish Stocks",
+    fill = "Status"
+  ) +
+  theme_minimal() +
+  theme(
+    plot.title = element_text(size = 16, face = "bold", hjust = 0.5),
+    plot.subtitle = element_text(size = 12, hjust = 0.5, margin = margin(b = 10)),
+    axis.text.x = element_text(angle = 45, hjust = 1, size = 10),
+    axis.title = element_text(size = 12, face = "bold"),
+    legend.title = element_text(size = 10, face = "bold"),
+    legend.text = element_text(size = 9),
+    strip.text = element_text(size = 12, face = "bold"),
+    panel.spacing = unit(1, "lines")
+  )
+plotB
+```
+
+This code combines two plots (`plotA` and `plotB`) into a single layout using `plot_grid`, arranging them side by side with two columns (`ncol = 2`). It then creates a title using `ggdraw` and `draw_label`, setting it with bold font, size 16, and centered alignment (`hjust = 0.5`). Finally, it combines the title and the plots into one final layout with `plot_grid`, stacking the title above the combined plots in a vertical arrangement, with the title taking 10% of the height and the plots taking the remaining 90%.
+
+```{r}
+combined_plot<- plot_grid(
+  plotA, plotB,
+  ncol = 2
+)
+title <- ggdraw() + draw_label("Fish Stock Status by Year Range and Ocean", fontface = 'bold', size = 16, hjust = 0.5)
+p5<-plot_grid(title, combined_plot, ncol = 1, rel_heights = c(0.1, 1))
+```
+
+## Final graph
+
+![Final graph](images/picture4.png){width="100%"}

```

Comment:
Instead of adding the static file, just print `p5` above, as you with the previous ones.